src/rendering/renderers/shared/system/SharedSystems.ts

```diff
@@ -25,14 +28,17 @@ export const SharedSystems = [
     TextureGCSystem,
     GenerateTextureSystem,
     ExtractSystem,
+    HTMLTextSystem
```

Comment:
This gets added in the `init.ts` for html text, so shouldn't be added here

Reply:
cool, removed!